shared/src/commonTest/kotlin/co/touchlab/kampstarter/BreedModelTest.kt

```diff
@@ -34,17 +33,17 @@ class BreedModelTest: BaseTest() {
     @Test
     fun staleDataCheckTest() = runTest {
         settings.putLong(BreedModel.DB_TIMESTAMP_KEY, currentTimeMillis())
-        assertFalse(ktorApi.jsonRequested)
+        assertTrue(ktorApi.mock.getJsonFromApi.calledCount == 0)
         model.getBreedsFromNetwork().join()
-        assertFalse(ktorApi.jsonRequested)
+        assertTrue(ktorApi.mock.getJsonFromApi.calledCount == 0)
     }
 
     @Test
     fun updateFavoriteTest() = runTest {
+        ktorApi.mock.getJsonFromApi.returns(ktorApi.successResult())
         model.getBreedsFromNetwork().join()
         itemDataSummary.await(500)
         val breedOld = dbHelper.selectAllItems().executeAsList().first()
-        assertEquals("appenzeller", breedOld.name)
```

Comment:
Removed because this breed was not the first on some platforms. I did change the setup of the result some, so that may be the cause. Ultimately we don't care which breed it is for this test, but it's curious

Reply:
I also noticed this when I was grabbing screenshots for my talk. Probably we can add an `ORDER BY` to the sql query to make it more consistent

Reply:
If we're losing the explicit breed name assert, does it make sense to at least assert that `breedOld` and `breedNew` have the same name? If the order started changing between calls it would break the assumptions of this test and you could potentially get some false positives.